src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -161,6 +166,34 @@ std::string ClassNamePrefix(const string& classname,
   return "";
 }
 
+std::string ClassNamePrefix(const string& classname,
+                            const EnumValueDescriptor* desc) {
```

Comment:
The value of `desc` is not used at all in this function. Perhaps this could be removed, and the function renamed to something like `ConstantNamePrefix`, since it is only used in that context.

Reply:
Done